src/code/dialogs/importDialog.js

```diff
@@ -26,8 +26,8 @@ const ImportDialog = WDialog.extend({
   removeHooks: function () {
     WDialog.prototype.removeHooks.call(this);
 
-    window.map.fire("wasabeeUIUpdate", { reason: "import" }, false);
-    window.map.fire("wasabeeCrosslinks", { reason: "import" }, false);
+    window.map.fire("wasabee:uiupdate:mapdata", { reason: "import" }, false);
```

Comment:
this can also be moved to the import part instead of `removeHooks`

Reply:
yes